src/components/uikit/Accordion.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from 'react';
+import { TouchableOpacity } from 'react-native';
+import Collapsible, { CollapsibleProps } from 'react-native-collapsible';
+
+import { Color, styled } from '~styles';
+
+import { Icon, IconName } from './Icon';
+import { Text } from './Text';
+import { Stack } from './layout/Stack';
+
+type AccordionProps = Omit<CollapsibleProps, 'collapsed'> & {
+  title: string;
+  initialOpen?: boolean;
+  children: React.ReactNode;
+  icon?: IconName | null;
+  iconColor?: Color | null;
+};
+
+export function Accordion({
+  title,
+  initialOpen = false,
+  children,
+  icon,
+  iconColor,
+  ...rest
+}: AccordionProps) {
+  const [isCollapsed, setCollapsed] = useState(!initialOpen);
+
+  return (
+    <Stack axis="y" spacing="small">
+      <TouchableOpacity onPress={() => setCollapsed((p) => !p)}>
+        <AccordionTitle
+          title={title}
+          icon={icon}
+          iconColor={iconColor}
+          isCollapsed={isCollapsed}
+        />
+      </TouchableOpacity>
+
+      <Collapsible {...rest} collapsed={isCollapsed}>
+        {children}
+      </Collapsible>
+    </Stack>
+  );
+}
+
+const Title = styled(Stack, {
+  borderBottomWidth: 1,
+  borderBottom: '$text',
```

Comment:
Should this be `borderBottomColor`? Also seems weird to use `$text` color here - would some neutral color make more sense?